packages/frontend/src/hooks/useEcharts.ts

```diff
@@ -494,9 +536,6 @@ const useEcharts = () => {
             top: 70, // pixels from top (makes room for legend)
             bottom: 30, // pixels from bottom (makes room for x-axis)
         },
-        ...(organisationData?.chartColors && {
```

Comment:
Why remove this?

Reply:
That was removed by mistake after a conflict U_U sorry